src/workerd/server/workerd.c++

```diff
@@ -591,6 +596,11 @@ public:
                           "<addr> instead of the address specified in the config file.")
         .addOptionWithArg({'i', "inspector-addr"}, CLI_METHOD(enableInspector), "<addr>",
                           "Enable the inspector protocol to connect to the address <addr>.")
+#if defined(WORKERD_USE_PERFETTO)
+        .addOptionWithArg({"p", "perfetto-trace"}, CLI_METHOD(enablePerfetto),
+                           "<path>=<categories>",
+                           "Enable perfetto tracing output to the specified file.")
```

Comment:
No action required: in the future, we might want to be able to provide a configuration TraceConfig

Reply:
Added a comment